### PR TITLE
Add historical release notes for v0.13.0

### PR DESCRIPTION
**Pull request by Michagogo**

### CODE REVIEW

File: doc/release-notes/release-notes-0.13.0.md [hunk 1]
```diff
@@ -0,0 +1,863 @@
...
+- #7814 `77b637f` Switch to py3 (MarcoFalke)
+- #8030 `409a8a1` Revert fatal-ness of missing python-zmq (laanwj)
+- #8018 `3e90fe6` Autofind rpc tests --srcdir (jonasschnelli)
+- #8016 `5767e80` Fix multithread CScheduler and reenable test (paveljanik)
+- #7972 `423ca30` pull-tester: Run rpc test in parallel  (MarcoFalke)
+- #8039 `69b3a6d` Bench: Add crypto hash benchmarks (laanwj)
+- #8041 `5b736dd` Fix bip9-softforks blockstore issue (MarcoFalke)
+- #7994 `1f01443` Add op csv tests to script_tests.json (Christewart)
+- #8038 `e2bf830` Various minor fixes (MarcoFalke)
+- #8072 `1b87e5b` Travis: 'make check' in parallel and verbose (MarcoFalke)
```
Comment (MarcoFalke) [inline doc/release-notes/release-notes-0.13.0.md]: Please use the one from master.

Concept ACK, otherwise.

**Review by maflcko — DISMISSED** [review]

**Comment by MarcoFalke** [comment]
ACK 431306f4d7e91c058499b4ee673f9f651643ea62

**Comment by btcdrak** [comment]
ACK

**Comment by fanquake** [comment]
ACK c9ffe90

**Pull request merged**